examples/coap-server.c

```diff
@@ -1276,15 +1319,17 @@ hnd_get(coap_context_t *ctx UNUSED_PARAM,
   resource_entry = &dynamic_entry[i];
 
   if (resource_entry->value) {
-    value.length = resource_entry->value->length;
-    value.s = resource_entry->value->s;
+    /* Bump reference so not removed elsewhere */
+    resource_entry->value->ref_cnt++;
+    value.length = resource_entry->value->value->length;
```

Comment:
See previous comment on why `resource_entry->value->value` might be NULL.

Reply:
Again, this is OK as `alloc_resource_data()` cannot return with` transient_value->value == NULL`.